src/main/webapp/group.js

```diff
@@ -106,8 +106,6 @@ async function getPosts(groupId, document, window) {
         var postsContainer = document.getElementById("posts");
 
         var numSearchResults = document.getElementById("num-posts-results");
-
-        postsContainer.appendChild(numSearchResults);
```

Comment:
Shouldn't this be kept?

Reply:
I added the results label H4 element to the HTML directly, so we can just get it by calling getElementById.